scripts/build/types/Theme.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface Theme {
```

Comment:
If we use TypeScript, TypeScript will automatically generate the type declaration files for us. This seems tedious having to manually create these.